ignite/contrib/handlers/tensorboard_logger.py

```diff
@@ -413,8 +414,7 @@ def __init__(self, *args, **kwargs):
         try:
             from tensorboardX import SummaryWriter
         except ImportError:
-            raise RuntimeError("This contrib module requires tensorboardX to be installed. "
-                               "Please install it with command: \n pip install tensorboardX")
+            from torch.utils.tensorboard import SummaryWriter
```

Comment:
Actually, thinking more about this, maybe we should warn user instead of `ImprotError` if it uses `torch<1.2.0`

Reply:
Yes, I've just updated the PR with what I tried to go further with testing the fallback, but this indeed may be sufficient.

Reply:
Here, I would like to cover quickly the case when users are using `torch==1.0.1`  for example:
```
# python -c "import torch; print(torch.__version__)"
1.0.1
# python -c "from torch.utils.tensorboard import SummaryWriter"
Traceback (most recent call last):
  File "<string>", line 1, in <module>
ModuleNotFoundError: No module named 'torch.utils.tensorboard'
```
Instead of `ModuleNotFoundError` we could inform user to install tensorboardX or upgrade torch to the latest.